sdk/include/aditof/camera.h

```diff
@@ -148,6 +148,20 @@ class SDK_API Camera {
     virtual Status requestFrame(Frame *frame,
                                 FrameUpdateCallback cb = nullptr) = 0;
 
+    /**
+     * @brief Scale AB image with logarithmic base 10.
+     * @param frame - The frame of the camera
+     * @param normalizeSteps - If is 0 then the algorithm is not applied,
+     * if is 1 is applied the normalization between 0 and 255,
+     * if is 2 is applied normalization between 0 and 255 and log10
```

Comment:
I think the function parameter can be improved in terms of simplicity.
The way I see it is that you're always doing the normalization (expect when 0 is provided which doesn't make sense. If one doesn't want the algorithm to be applied he just won't call the method) so it would be simpler to change the 'normalizeSteps' to something like 'useLogScaling'. When 'useLogScaling' = true, the normalization between 0 and 255 is applied together with log10. When 'useLogScaling' = false, just the normalization is being done (as the name of the method suggests).